src/drivers/imx/ipc.c

```diff
@@ -115,6 +115,7 @@ void ipc_platform_complete_cmd(struct ipc *ipc)
 
```

Comment:
I believe there are 2 issues fixed here: PM not supported on imx93 and MU version 2 - so, I would split this in 2 patches.

Reply:
ACK, will fix.